wye/social/sites/twitter.py

```diff
@@ -0,0 +1,34 @@
+from twython import Twython
+
+from django.conf import settings
+
+from wye.social.utils import get_message
+
+
+def init_twitter():
+    try:
+        twitter = Twython(
+            settings.TWITTER_CONSUMER_KEY,
+            settings.TWITTER_CONSUMER_SECRET,
+            settings.TWITTER_ACCESS_TOKEN,
+            settings.TWITTER_ACCESS_TOKEN_SECRET,
+        )
+    except:
```

Comment:
Both the try except blocks are required in this case. I'll remove the else. Do we have a logger for the project already?